scenario_tests/collations/test.sh

```diff
@@ -8,17 +8,23 @@ source ../test-env.sh
 # Run service
 docker-compose up -d
 
+if [[ -n "${PRINT_TEST_LOGS}" ]]; then
+  docker-compose logs -f &
+fi
+
```

Comment:
All the scenarios must have this so we can see the log in GH Action log as the tests runs asynchronously.